prow/cmd/deck/main.go

```diff
@@ -80,6 +81,7 @@ type options struct {
 	jobConfigPath         string
 	buildCluster          string
 	kubernetes            prowflagutil.ExperimentalKubernetesOptions
+	github                prowflagutil.GitHubOptions
```

Comment:
Yes, it's ok to do this, there's no way around it. We want to update `deck` to point at `ghproxy` as well in the deployment